src/lib/minions/data/killableMonsters/custom/AbyssalDragon.ts

```diff
@@ -0,0 +1,107 @@
+import { Monsters } from 'oldschooljs';
+import { RareTable } from 'oldschooljs/dist/simulation/clues/Beginner';
+import { GemTable } from 'oldschooljs/dist/simulation/subtables/RareDropTable';
+import TreeHerbSeedTable from 'oldschooljs/dist/simulation/subtables/TreeHerbSeedTable';
+import WyvernHerbTable from 'oldschooljs/dist/simulation/subtables/WyvernHerbTable';
+import LootTable from 'oldschooljs/dist/structures/LootTable';
+
+import setCustomMonster from '../../../../util/setCustomMonster';
+
+export const AbyssalDragonLootTable = new LootTable()
+	.every('Abyssal dragon bones', 2)
+
+	// Uniques
+	.tertiary(
+		1,
+		new LootTable({ limit: 1 })
+			.add('Abyssal lance', 1, 1 / 5120)
+			.add('Abyssal defender', 1, 1 / 5120)
+			.add('Abyssal cape', 1, 1 / 5120)
+
+			.add('Ori', 1, 1 / 3076)
+			.add('Dragcula', 1, 1 / 3076)
+			.add('Mally', 1, 1 / 3076)
+
+			.add('Abyssal thread', 1, 1 / 1024)
+
+			.add('Dragon hunter lance', 1, 1 / 512)
+			.add('Dragon hunter crossbow', 1, 1 / 512)
+	)
+
+	// Uniques
+	.tertiary(
+		2,
+		new LootTable()
+			.add('Clue scroll (master)', [1, 2], 1)
+			.add('Clue scroll (master)', 1, 2)
+
+			.add('Clue scroll (elite)', [1, 3], 2)
+			.add('Clue scroll (elite)', 1, 4)
+
+			.add('Clue scroll (hard)', [1, 4], 4)
+			.add('Clue scroll (hard)', 1, 8)
+	)
+
+	.oneIn(
+		30,
+		new LootTable()
+			.add('Black dragonhide', [100, 200])
+			.add('Red dragonhide', [100, 200])
+			.add('Blue dragonhide', [100, 200])
+			.add('Green dragonhide', [100, 200])
+			.add('Coal', [20, 140])
+			.add('Yew logs', 500)
+			.add('Magic logs', 250)
+			.oneIn(2, 'Gold ore', 600)
+			.oneIn(10, 'Raw shark', 200)
+			.oneIn(15, 'Raw manta ray', 200)
+			.oneIn(512, 'Dragon med helm', 73)
+	)
+
+	.oneIn(
+		1000,
+		new LootTable()
+			.add('Draconic visage')
+			.oneIn(2, 'Skeletal visage')
+			.oneIn(3, 'Wyvern visage')
```

Comment:
this is a really weird way to write the table...you should use just normal weightings here